config/locales/dotiw/en.yml

```diff
@@ -2,4 +2,4 @@
 en:
   datetime:
     dotiw:
-      two_words_connector: and
+      two_words_connector: ' and '
```

Comment:
I feel like we go back and forth on the spaces in this one all the time...?

Reply:
> I feel like we go back and forth on the spaces in this one all the time...?

Heh, I had a feeling I'd seen this before as well. Let me see if I can dig up any relevant history.

Reply:
Ah, I think we've seen it recently because we tend to overwrite this from the call to `distance_of_time_in_words`.

For example:

https://github.com/18F/identity-idp/blob/5d12301efa96f715ec0a1e6fb763502a633ee521/app/helpers/session_timeout_warning_helper.rb#L21-L27

We could do the same here, although it kinda feels like it might be better to tackle it at the source? We'd want to update these other overrides if we do, otherwise we'll get some "3 hours&nbsp;&nbsp;and&nbsp;&nbsp;2 minutes" excessive spacing issues.

Reply:
For reference, the default is " and ", with the spaces:

https://github.com/radar/distance_of_time_in_words#two_words_connector

I don't know that we had any strong reason to omit the spaces.

Reply:
I think that part of the issue was a fight of custom strings plus our limited YAML normalization at the time (removing trailing spaces, etc), but I agree seems better to update at the source

Reply:
Yep, I think that's exactly right (https://github.com/18F/identity-idp/pull/3780#discussion_r431246132). I'd like to think our fancy new YAML normalizer can handle this now. It's worth a shot!

Reply:
This was addressed separately in #5312 and dropped in the rebased bd57718.